src/partcad/project.py

```
@@ -188,6 +214,20 @@ def __enter__(self, *_args):
         def __exit__(self, *_args):
             self.lock.release()
 
+    class SoftwareLock(object):
+        def __init__(self, prj, software_name: str):
+            prj.software_locks_lock.acquire()
+            if not software_name in prj.software_locks:
```

Ruff (0.16.2)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)